django-allauth-0.39.0-py_0
